receiver/awsecscontainermetricsreceiver/awsecscontainermetrics/metrics_helper.go

```diff
@@ -26,6 +26,7 @@ func getContainerMetrics(stats ContainerStats) ECSMetrics {
 	netStatArray := getNetworkStats(stats.Network)
 
 	storageReadBytes, storageWriteBytes := extractStorageUsage(&stats.Disk)
+	floatZero := float64(1.0)
```

Comment:
The variable name is `Zero` but why it has value 1? 
And why the metric value is 1 when the datasource is absent?

Reply:
good catch. It should be zero.